tests/samples/expected/ocaml.mli

```diff
@@ -5372,15 +5375,15 @@ module Token: sig
   val transfer_n :
     ?origin: Receipt.update_origin ->
     context ->
-    ([<source] * Tez.t) list ->
-    [<sink] ->
+    ( [< source] * Tez.t) list ->
```

Comment:
Should there be a space here, after the open paren?
```suggestion
    ([< source] * Tez.t) list ->
```

Reply:
(By "should", I mean, "is this what we want", rather than "is this what we expect".)

Reply:
There shouldn't, well spotted